src/main/java/com/smartsheet/api/FolderResources.java

```diff
@@ -48,7 +49,7 @@
 	 * @throws ServiceUnavailableException if the REST API service is not available (possibly due to rate limiting)
 	 * @throws SmartsheetException if there is any other error during the operation
 	 */
-	public Folder getFolder(long folderId) throws SmartsheetException;
+	public Folder getFolder(long folderId, EnumSet<FolderInclusion> includes) throws SmartsheetException;
```

Comment:
Hey Brett, I have corrected and used sourceInclusion. Deleted FolderInclusion class from project.